core/spec/lib/factory_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe FactoryBot do
+  it { FactoryBot.lint(FactoryBot.factories.reject{ |f| f.name == :customer_return_without_return_items }) }
```

Comment:
Can you add a description to this `it` block?

Reply:
Of course! Hope the description I added is clear enough.